distributed/tests/test_resources.py

```diff
@@ -277,9 +277,10 @@ def test_full_collections(c, s, a, b):
 
 
 @pytest.mark.parametrize('optimize_graph', [
-    pytest.mark.xfail(True, reason="don't track resources through optimization"),
-    pytest.mark.skipif(WINDOWS, False, reason="intermittent failure"),
-])
+    pytest.param(True, marks=[
```

Comment:
Not 100% on this syntax for combining multiple marks. It's marks with an "S", but I cannot find docs/example of more than one.

Reply:
This looks correct. You can pass a single mark (not wrapped in a list) to `marks` as well.

Reply:
It was originally two tests cases I misread and combines. Fixed in latest push.